### ISSUE TITLE
NonMaxSuppression in model causes onnx_tool exception

### ISSUE THREAD

**Issue by verderog** [ThanatosShinji/onnx-tool#69]
**Describe the bug**
I encounter an error when trying to profile this model: **yolox_s_lite_640x640_20220221_model.onnx** (see below for download link)

```
$ python -m onnx_tool -i yolox_s_lite_640x640_20220221_model.onnx --mode profile
Traceback (most recent call last):
  File "/home/user/.pyenv/versions/3.8.10/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/home/user/.pyenv/versions/3.8.10/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/home/user/.pyenv/versions/3.8.10/lib/python3.8/site-packages/onnx_tool/__main__.py", line 93, in <module>
    onnx_tool.model_profile(args.in_, dynamic, args.file, args.out)
  File "/home/user/.pyenv/versions/3.8.10/lib/python3.8/site-packages/onnx_tool/__init__.py", line 84, in model_profile
    g.shape_infer(dynamic_shapes)
  File "/home/user/.pyenv/versions/3.8.10/lib/python3.8/site-packages/onnx_tool/graph.py", line 829, in shape_infer
    node.shape_infer(itensors, otensors)
  File "/home/user/.pyenv/versions/3.8.10/lib/python3.8/site-packages/onnx_tool/node.py", line 551, in shape_infer
    max_output_boxes_per_class = int(intensors[2].get_numpy()[0])
IndexError: too many indices for array: array is 0-dimensional, but 1 were indexed
```

I'm not certain if this is a model problem or a tool problem.  The `netron` tool is able to process this model just fine and running `onnx.checker.check_model` doesn't report any errors, so I suspect it's `onnx-tool`.  

**To Reproduce**
1. Download http://software-dl.ti.com/jacinto7/esd/modelzoo/09_01_00/models/vision/detection/coco/edgeai-mmdet/yolox_s_lite_640x640_20220221_model.onnx
2. python -m onnx_tool -i yolox_s_lite_640x640_20220221_model.onnx --mode profile
3. See error

**Expected behavior**
Either fail gracefully or correctly process NMS parms.

**Comment by ThanatosShinji** [comment]
fixed in 116d00ef646dc0ac8795c109baa8b9bf605e1128
Your model has been verified in the main branch code.

**Comment by verderog** [comment]
Great!  I just upgraded to the latest version and confirmed it works for my model.

Thanks for the quick turnaround!
